config/routes.rb

```diff
@@ -1,5 +1,5 @@
 Rails.application.routes.draw do
-  root to: redirect('/is_it_working')
+  root to: redirect('/status/all')
```

Comment:
I figured `/status/all` will provide us with the best overview of what's going on (and would be the equivalent to the `/is_it_working` route).